website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```diff
@@ -174,20 +174,31 @@ You can connect to external Iceberg-compatible catalogs, such as Polaris and Uni
 
 These are the additional configurations, unique to Snowflake, that can be supplied and nested under `adapter_properties`. 
 
+#### Built-In Catalog
+
 | Field | Required | Accepted Values |
 | --- | --- | --- |
-| `storage_serialization_policy` | Optional | `COMPATIBLE` or `OPTIMIZED`     |
-| `max_data_extension_time_in_days` | Optional |  `0` to `90` with a default of `14`  |
-| `data_retention_time_in_days` | Optional | Standard Account: `1`, Enterprise or higher: `0` to `90`, default `1`  |
 | `change_tracking` | Optional | `True` or `False`    |
+| `data_retention_time_in_days` | Optional | Standard Account: `1`, Enterprise or higher: `0` to `90`, default `1`  |
+| `max_data_extension_time_in_days` | Optional |  `0` to `90` with a default of `14`  |
+| `storage_serialization_policy` | Optional | `COMPATIBLE` or `OPTIMIZED`     |
+
+#### Rest Catalog
+
+| Field | Required | Accepted Values |
+| --- | --- | --- |
+| `auto_refresh` | Optional | `True` or `False`    |
 | `catalog_linked_database` | Required if you are using the iceberg_rest `catalog type`. | catalog linked database name.   |
+| `max_data_extension_time_in_days` | Optional |  `0` to `90` with a default of `14`  |
+| `target_file_size` | Optional | values like `'AUTO'`, `'16MB'`, `'32MB'`, `'64MB'`, `'128MB'`, etc., case insensitive  |
 
 -  **storage_serialization_policy:** The serialization policy tells Snowflake what kind of encoding and compression to perform on the table data files. If not specified at table creation, the table inherits the value set at the schema, database, or account level. If the value isn’t specified at any level, the table uses the default value. You can’t change the value of this parameter after table creation.
```

Comment:
just curious - what do these fields (lines 195-201) mean or used for? Are they also REST catalog fields?